lib/tasks/uploads.rake

```diff
@@ -597,13 +597,13 @@ task "uploads:disable_secure_uploads" => :environment do
     puts "Disabling secure upload and resetting uploads to not secure in #{db}...", ""
 
     SiteSetting.secure_uploads = false
+    SiteSetting.secure_uploads_pm_only = false
 
     secure_uploads =
       Upload
-        .joins(:upload_references)
-        .where(upload_references: { target_type: "Post" })
+        .joins("LEFT JOIN upload_references ON uploads.id = upload_references.upload_id")
```

Comment:
Do we still need that join? I think that because it is left join, if you want to find uploads with upload_reference, then you need additional check like  `WHERE upload_references.id IS NOT NULL`. Or maybe we don't care if there is a link, then we can drop join completely.

Reply:
Oh yeah good point :thinking: I guess it should be ` WHERE upload_references.id IS NOT NULL`, you are right

Reply:
Actually no, even with ones with zero references we want to un-secure them, since that will allow them to get destroyed by our upload cleanup, will remove the join